epay3.Web.Api.Tests/App.QA.config

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0"?>
+<!-- For more information on using app.config transformation visit    http://go.microsoft.com/fwlink/?LinkId=125889 -->
+<configuration xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">
+ <appSettings>
+  <add key="ApiUri" value="https://api.epay3qa.com" xdt:Transform="Replace" xdt:Locator="Match(key)"/>
+  <add key="ApiKey" value="a3d14692344443e" xdt:Transform="Replace" xdt:Locator="Match(key)"/>
```

Comment:
Hmmm..this key/secret combination isn't working for me on QA for some reason?

Reply:
Looks like it got regenerated :(